doc/langdef.md

```diff
@@ -157,7 +157,6 @@ ESCAPE         ::= \ [abfnrtv\?"'`]
                  | \ u HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT
                  | \ U HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT HEXDIGIT
                  | \ [0-3] [0-7] [0-7]
-NEWLINE        ::= \r\n | \r | \n
```

Comment:
Seemed better to remove this since different contexts use different definitions of a newline.